packages/passport-auth/lib/Passport.js

```diff
@@ -0,0 +1,306 @@
+const passport = require('passport');
+
+const FIELD_SERVICE_NAME = 'service';
+const FIELD_USER_ID = 'serviceUserId';
+const FIELD_USERNAME = 'serviceUsername';
+const FIELD_TOKEN_SECRET = 'tokenSecret';
+const FIELD_ITEM = 'item';
+
+let isInitialized = false;
+
+class PassportAuthStrategy {
+  constructor(authType, keystone, listKey, config, ServiceStrategy = null) {
+    this.authType = authType;
+    this.keystone = keystone;
+    this.listKey = listKey;
+    this.config = {
+      tokenSecretField: FIELD_TOKEN_SECRET,
+      itemField: FIELD_ITEM,
+      sessionListKey: `passportSession`,
+      useSession: false,
+      sessionIdField: 'passport',
+      keystoneSessionIdField: 'keystone_passport',
+      scope: [],
+      ...config,
+    };
+    this.ServiceStrategy = ServiceStrategy;
+
+    this.createSessionList();
+
+    PassportAuthStrategy.initializePassport(this.config.server);
+
+    this.passportStrategy = this.getPassportStrategy();
+    passport.use(this.passportStrategy);
+  }
+
+  static initializePassport(server) {
```

Comment:
`initializePassport` should be removed, and the onus should be on the system builder to call `server.app.use(passport.initialize());` at the appropriate place in their middleware stack.

In the future we can add helper functions around that, but we shouldn't be forcing our way into the middleware stack right here.

Removing this method means we no longer need to pass in a `server` to `PassportAuthStrategy`.

Reply:
added a helper function to `PassportAuthStrategy` which is exposed on `@keystone-alpha/passport-auth`